assets/blocks/course-progress/settings.js

```diff
@@ -0,0 +1,87 @@
+import { InspectorControls } from '@wordpress/block-editor';
+import { PanelBody, PanelRow, RangeControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * The course progress block settings.
+ *
+ * @param {Object}   props                     Component properties.
+ * @param {number}   props.manualPercentage    The value of the percentage.
+ * @param {Function} props.setManualPercentage Callback to set the value of manual percentage.
+ * @param {number}   props.borderRadius        The value of the bar radius.
+ * @param {Function} props.setBorderRadius     Callback to set the value of border radius.
+ * @param {number}   props.height              The value of the bar height.
+ * @param {Function} props.setHeight           Callback to set the value of height.
+ */
+export function CourseProgressSettings( {
+	manualPercentage,
+	setManualPercentage,
+	borderRadius,
+	setBorderRadius,
+	height,
+	setHeight,
+} ) {
+	const initialHeight = 14;
+	const initialBorderRadius = 2;
+
+	borderRadius =
```

Comment:
Instead of these initial values and checks, do you think it would make sense to add a default value for the attributes?

Reply:
I thought about this too but then this would mean that there will always be an inline attribute for the attributes and it would make it harder to override in CSS if the user whishes so.

Reply:
Ah, I got your point!